lib/matplotlib/table.py

```diff
@@ -475,17 +514,11 @@ def auto_set_column_width(self, col):
         0: the 1st column
```

Comment:
the previous line should read "labeling" (not labling)

Reply:
Docstring rewritten and explanation of indices removed. (Also it's not true: If there are labels, 0 is the label row/column; if not 0 is the first row/column). This is a (IMHO not carefully chosen implementation detail of `table()` which I do not want to discuss in the docstring of `Table`.)